server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsResponse.java

```diff
@@ -87,4 +91,40 @@ public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params par
         return builder;
     }
 
+    public static GetSnapshotsResponse fromXContent(XContentParser parser) throws IOException {
```

Comment:
any reason that an object parser was not used here?

Reply:
Changed